### PR TITLE
Removed unused fwkJobReports from MessageLogger in HLTrigger Subsystem

### PR DESCRIPTION
**Pull request by Dr15Jones**
#### PR description:

The parameter fwkJobReports is an obsolete parameter which has had no functionality for many years.
The new customize function removes the parameters if present.

#### PR validation:

The new customize function was tested manually both with and without the parameters and shown to work properly.

### CODE REVIEW

**Comment by Dr15Jones** [comment]
please test

File: HLTrigger/Configuration/python/customizeHLTforCMSSW.py [hunk 1]
```diff
@@ -203,10 +203,22 @@ def customiseFor2018Input(process):
     process = synchronizeHCALHLTofflineRun3on2018data(process)
 
 
+def customizeToDropObsoleteMessageLoggerOptions(process):
```
Comment (Martin-Grunewald) [inline HLTrigger/Configuration/python/customizeHLTforCMSSW.py:206]: Please add the PR number in the name or in a comment.

**Comment by Martin-Grunewald** [comment]
@Dr15Jones 
Could you then also remove it here:
https://github.com/cms-sw/cmssw/blob/master/FWCore/MessageService/python/MessageLogger_cfi.py#L87
which is where ConfDB piscks up this parameter.

File: HLTrigger/Configuration/python/customizeHLTforCMSSW.py (hunk 1 above)
Comment (Dr15Jones) [reply HLTrigger/Configuration/python/customizeHLTforCMSSW.py:206]: There is no pull request yet which requires this change (that will only happen after all these changes go through), or do you mean you want the pull request number for this pull request?

**Comment by Dr15Jones** [comment]
> Could you then also remove it here:
https://github.com/cms-sw/cmssw/blob/master/FWCore/MessageService/python/MessageLogger_cfi.py#L87
which is where ConfDB picks up this parameter when parsing a new release.

The plan is to modify that once all the pull requests which were using it are accepted.

File: HLTrigger/Configuration/python/customizeHLTforCMSSW.py (hunk 1 above)
Comment (Martin-Grunewald) [reply HLTrigger/Configuration/python/customizeHLTforCMSSW.py:206]: Usually it is this PRs number, to remind why this customisation was introduced when I remove it later after a new release has been parsed into ConfDB.

**Comment by Martin-Grunewald** [comment]
Is that PR coming soon? ie, before/in time for pre8?

**Comment by Dr15Jones** [comment]
> Is that PR coming soon? ie, before/in time for pre8?

That primarily depends on how quickly all the pull requests get signed. Historically, some areas are not as quick as others to sign PRs.

**Comment by Martin-Grunewald** [comment]
Uhhmm then I need to carry this customisation around ....

**Comment by Martin-Grunewald** [comment]
please test

**Comment by Martin-Grunewald** [comment]
+1

**Comment by silviodonato** [comment]
+1

**Pull request merged**